python/test_opensky_api.py

```diff
@@ -101,6 +101,61 @@ def test_state_vector_parsing(self):
         self.assertEqual(0, s.position_source)
         self.assertEqual(0, s.category)
 
+    def test_flight_data_parsing(self):
+        keys = [
+            "icao24",
+            "firstSeen",
+            "estDepartureAirport",
+            "lastSeen",
+            "estArrivalAirport",
+            "callsign",
+            "estDepartureAirportHorizDistance",
+            "estDepartureAirportVertDistance",
+            "estArrivalAirportHorizDistance",
+            "estArrivalAirportVertDistance",
+            "departureAirportCandidatesCount",
+            "arrivalAirportCandidatesCount",
+        ]
+        values = ['4951d0', 1517230550, 'EDDF', 1517240237, 'LPCS', 'TAP583  ', 3808, 80, 14016, 708, 2, 3]
+        f = FlightData(values)
+        for key, exp_val in zip(keys, values):
+            self.assertEqual(exp_val, getattr(f, key))
+
+    def test_waypoint_parsing(self):
+        keys = [
+            "time",
+            "latitude",
+            "longitude",
+            "baro_altitude",
+            "true_track",
+            "on_ground",
+        ]
+        values = [1675450754, 39.0912, -94.5794, 304, 91, False]
+        w = Waypoint(values)
+        for key, exp_val in zip(keys, values):
+            self.assertEqual(exp_val, getattr(w, key))
+
+    def test_flight_track_parsing(self):
+        entry = {'icao24': 'a01391',
+                 'callsign': 'N104AA  ',
+                 'startTime': 1675450754.0,
```

Comment:
is the time in the API response float or int?

Reply:
Good catch, `int` is the correct type here. 
Done. I fixed types also in waypoint representation